tomography_preprocessing/tilt_series_alignment/aretomo/batch.py

```diff
@@ -41,6 +41,15 @@ def batch_aretomo(
     -------
 
     """
+    #Check input paths exist
+    if not Path.exists(tilt_series_star_file):
+        e = 'Could not find tilt series star file'
+        console.log(f'ERROR: {e}')
+        raise RuntimeError(e)
+    if aretomo_executable != None and not Path.exists(aretomo_executable):
```

Comment:
super minor but slight preference for `if aretomo_executable is not None`

I don't think the path existence check actually works as written - just tested and got 

```python
In [1]: from pathlib import Path

In [2]: Path.exists('.')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-49cc26ef6a68> in <module>
----> 1 Path.exists('.')

/opt/homebrew/Caskroom/miniforge/base/envs/py39/lib/python3.9/pathlib.py in exists(self)
   1412         """
   1413         try:
-> 1414             self.stat()
   1415         except OSError as e:
   1416             if not _ignore_error(e):

AttributeError: 'str' object has no attribute 'stat'
```

Instead it should be something like

```python
Path(aretomo_executable).exists()
```

the intuition being that you create the proper `Path` object then call the method which checks whether it exists. The syntax you wrote is reminiscent of a [`classmethod`/`staticmethod`](https://stackoverflow.com/questions/12179271/meaning-of-classmethod-and-staticmethod-for-beginner) so not totally out there and might be something you have come across, just doesn't work in this particular setting

Reply:
also, `Typer` will take care of formatting CLI args as `Path` objects so we should be able to get away with 

```python
if aretomo_executable is not None and aretomo_executable.exists()
```

You might have worried about whether this would evaluate properly if `aretomo_executable` was `None`... interesting language quirk that the left hand side is evaluated first, right hand side is never called if left hand side fails in an `and` comparison
```python
In [3]: aretomo_executable = None

In [4]: aretomo_executable is not None and aretomo_executable.exists()
Out[4]: False

In [5]: aretomo_executable = Path()

In [6]: aretomo_executable is not None and aretomo_executable.exists()
Out[6]: True
```